docs/index.rst

```diff
@@ -55,7 +55,7 @@ to automatically check every converter with
     from onnxmltools import convert_sklearn
     from onnxmltools.utils import save_model
     from onnxconverter_common.data_types import FloatTensorType
-    initial_type = [('float_input', FloatTensorType([1, 4]))]
```

Comment:
`1` will not work here, you need to supply the batch size, but hardcoding it at 38 (from the iris dataset) may break later if the dataset changes, so `-1` supplies a the batch dimension.